src/routes/Onboarding/CreateAccount.tsx

```diff
@@ -35,9 +22,11 @@ interface CreateAccountProps {
 
 const CreateAccount: FC<CreateAccountProps> = ({
   desktopMode = true,
-  onCreate,
+  onCreate = () => ({}),
```

Comment:
Default value is not a VoidFunction. Try to change 
```suggestion
  onCreate = () => {}, // or use noop from lodash
```

Reply:
replaced with undefined